cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/utils/ChaCha20Utils.kt

```diff
@@ -70,7 +70,7 @@ internal class ChaCha20Utils {
                 outputBufferedSink.write(encryptedData.toByteArray())
                 encryptedDataSize += encryptedData.size
             }
-        } catch (e: Exception) {
+        } catch (e: SecretStreamCorruptedOrTamperedDataException) {
```

Comment:
are we sure we want to propagte any other exceptions ?

Reply:
Yes, well thought. I will add other specific exceptions as well.